ui/src/utils/env.ts

```diff
@@ -0,0 +1,47 @@
+export {}
+
+const GIT_SHA =
+  process.env.INFLUXDB_SHA ||
```

Comment:
presumably this is so we can override the current git hash with an envvar?

Reply:
or define it externally through a CI pipeline that only has readonly access of the code, but yeah!